src/auth/Auth.ts

```diff
@@ -84,18 +87,25 @@ class Auth {
   /**
    * Updates the current session with a new access token. On error,
    */
-  public renewSession = (): void => {
-    this.auth0.checkSession({}, (err, authResult) => {
-      if (authResult && authResult.accessToken && authResult.idToken) {
-        this.setSession(authResult);
-      } else if (err) {
-        this.logout();
-        console.log(err);
-        alert(
-          `Could not get a new token (${err.error}: ${err.errorDescription}).`
-        );
-      }
-    });
+  /**
+   * Updates the current session with a new access token. On error,
+   */
+  public renewSession = async (): Promise<void> => {
+    return new Promise((resolve, reject) =>
+      this.auth0.checkSession({}, (err, authResult) => {
+        if (authResult && authResult.accessToken && authResult.idToken) {
+          this.setSession(authResult);
+          resolve();
+        } else if (err) {
+          this.logout();
+          console.log(err);
```

Comment:
Is it safe to log those errors?

Reply:
Hm good point. Since this is the error object received from the callback when renewing a token, this should be safe (I assume auth0 is not sending back any sensitive material). But honestly speaking, the current state there is not very end-user friendly, rather for "debugging"... What do you think? Just ignore the errors? Or should we add something like sentry for monitoring?

Reply:
Yes, sentry looks nice. I think the free developer tier should be enough for us? Otherwise, it will be quite expensive. But such a monitoring tool is a must-have in my opinion. Then we could just add a generic error message for the user like "Authentication failed, try again or contact the administrator". And send the full error to our monitoring system.

If ok for you we will delete the console log and replace the alert message for the user with a generic one. Then merge this and create a new issue to add sentry. We can also use sentry for our backend, right?

Reply:
yes - I agree. I have removed the console.log's, merge this and create new issues